controllers/policyserver_controller.go

```diff
@@ -57,6 +57,8 @@ type PolicyServerReconciler struct {
 //+kubebuilder:rbac:namespace=kubewarden,groups=apps,resources=deployments,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:namespace=kubewarden,groups=apps,resources=replicasets,verbs=get;list;watch
 //+kubebuilder:rbac:namespace=kubewarden,groups=core,resources=pods,verbs=get;list;watch
+//+kubebuilder:rbac:namespace=kubewarden,groups=policy,resources=poddisruptionbudgets,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=policy,resources=poddisruptionbudgets,verbs=get;list;watch;create;update;patch;delete
```

Comment:
We don't need the second line, only the first one. The first one grants privileges against PDB objects inside of a namespace. The 2nd one grants provileges against PDB resources inside of the whol cluster. This is too wide.

Reply:
I'm not sure about this. I had to add this line to make the  controller able to access the PDB. But I'll double check that.

Reply:
Testing this I found that some roles in the helm charts does not correspond to the inline annotations heres. Let me try to fix them.

Reply:
I've fixed this. I forgot to add the PDB in the client cache configuration. That's why I need the clusterscope permission.